examples/research_projects/rag/finetune_rag.py

```diff
@@ -74,27 +73,19 @@ def __init__(self, *args, **kwargs):
         self.__dict__ = self
 
 
-# In PTL >v1.0, `init_ddp_connection` method in the `LightningModule`
-# is no longer used, and is moved into DDPAccelerator instead.
-# We override DDPAccelerator to add our custom logic for initializing the
-# retriever.
-# https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/backends/test_accelerator_connector.py
-
-
-class CustomAccel(DDPAccelerator):
-    def __init__(self, trainer=None, **kwargs):
-        # Trainer is set later.
-        super().__init__(trainer, **kwargs)
+class MyDDP(DDPPlugin):
```

Comment:
not a big fan of the class name `MyDDP` -> couldn't we just keep `CustomAccel`?

Reply:
Yeah got it. The new PL version has two different pluggings to Accelerators and DDP (both a still in experiment though). So to make it clear. I renamed it **CustomDDP**.

Reply:
ok for me since we have to break backward compatibility anyways